### ISSUE TITLE
Github warning: `set-output` command is deprecated

### ISSUE THREAD

**Issue by boozook** [MarceloPrado/has-changed-path#34]
```
Warning: The `set-output` command is deprecated and will be disabled soon.
Please upgrade to using Environment Files. 
For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

**Comment by boozook** [comment]
Fixed in #35.

**Comment by cyfung1031** [comment]
@boozook this issue shall be no longer valid since 

<img width="473" alt="Screen Shot 2023-11-30 at 21 00 29" src="https://github.com/MarceloPrado/has-changed-path/assets/44498510/8f0f8a6d-838f-4179-bf7e-d7a4a378aa8f">

<img width="1309" alt="Screen Shot 2023-11-30 at 21 00 39" src="https://github.com/MarceloPrado/has-changed-path/assets/44498510/9d1e46a0-8ea0-461c-8fd2-7c8bf1f2479c">

You shall update your workflow from `marceloprado/has-changed-path@v1` to `marceloprado/has-changed-path@v1.0.1`

This issue shall be closed.
